README.md

```diff
@@ -273,22 +275,27 @@ It is also recommended but not required that each bot have a unique command pref
 
 ## Appendix
 
-### config.yaml parameter descriptions
+### Parameter Descriptions for config.yaml
 
-#### Core bot parameters
+#### Core Bot Parameters
 
--   **name** - A name for the bot. It can be unique if you want, but is not required. However it will be more difficult to distingush multiple bots if you are running multiple.
--   **token** - A discord developer API token. You can get one [here](https://discord.com/developers/applications)
--   **command_prefix** - A single character which you will use to preface all commands for this bot e.g. !music
--   **logging** - Enable or disable logging for this bot by setting the enabled value to True or False.
--   -   **logging_level *(Default=NOTSET)*** - The level of logging. Possible options: NOTSET, INFO, WARNING, ERROR, DEBUG
+|Parameter|Default Value(s)|Description|
+|---|:---:|---|
+|name|`null`|A name for the bot. It can be unique if you want, but is not required. However it will be more difficult to distingush multiple bots if you are running multiple.|
+|token|`null`|A discord developer API token. You can get one [here](https://discord.com/developers/applications)
+|command_prefix|`"!"`|A single character which you will use to preface all commands for this bot e.g. `!music`|
+|logging|`NOTSET`|The level of logging. Possible options: NOTSET, INFO, WARNING, ERROR, DEBUG
 
-#### Music Feature
 
--   **local_path** - A system path pointing to a local library of music. Ideally the directory structure will be **local_path/Artists/Albums/songs** But any structure should work. The music feature will search this directory for songs which match the **audio_types** every **search_frequency** seconds.
--   **search_frequency** - How frequently you want the music feature to search for new music. This value is in seconds. Default is 300 seconds (5 minutes)
--   **audio_types** - The different audio formats you want the music feature to search for. These audio types must be readable by ffmpeg.
+#### Music Feature
+|Parameter|Default Value(s)|Description|
+|---|:---:|---|
+|local_path|`/music`|A system path pointing to a local library of music. Ideally the directory structure will be `local_path/Artists/Albums/songs` But any structure should work. The music feature will search this directory for songs which match the **audio_types** every **search_frequency** seconds.|
```

Comment:
Do you notice how in the formatted table, the default values column, each default value alternates a grey background?
[example](https://i.imgur.com/hEWcJMB.png)

I wonder if it is [this](https://stackoverflow.com/questions/50302165/disable-zebra-striping-in-github-table-markdown)

Anyway, I personally think it would look better either all of them greyed, or none of them.

Reply:
I attempted to look into this and there doesn't seem to be a way of handling this without causing the table to look worse. Case-in-point, adding an empty row: 
![image](https://user-images.githubusercontent.com/43051972/97758375-bf4bd280-1acc-11eb-9e20-e43ace0f35a9.png)